
syntax = "2023";
package telir;

message SourceFile {
  string name = 1;
  string source = 2;
}

